parsing_minmax.py

```diff
@@ -0,0 +1,61 @@
+##Developing a Database that tells us the min&max repeat sizes in samples for each locus
+#Columns: repeat	min_size	max_size	min_size_sample	max_size_sample (5 columns)
+
+##Steps
+#Iterate over each line and group everything by chromosome loci
+#for each allele (2 of them per sample), print out the length
+#For each loci, extract the min and max allele lengths, along with their sampleIDs (5 values, loci, min, max, sampleIDmin, sampleIDmax)
+#Create a dataframe of this information (pandas) and sort them based on chromosome location (ascending order)
+
+
+import gzip
+import itertools
+import numpy as np
+import pandas as pd
+from collections import namedtuple
+
+RepeatRec = namedtuple("RepeatRec", "sample short_allele long_allele")
+
+#Parse through file and extract the repeat information (sample, alleles' size) along with their chrom location
+def get_repeat_recs(path):
+    def parse_alleles(group):
+        alleles = list(line.decode("utf8").split() for line in group)
+        alleles = [(rec[1], rec[2].split(",")) for rec in alleles]
+        return alleles
+    
+    with gzip.open(path, "r") as file:
+        for trid, group in itertools.groupby(file, key=lambda line: line.decode("utf8").split()[0]):
+            alleles = parse_alleles(group)
+            repeat_recs = [(s, [len(a) for a in als]) for s, als in alleles]
```

Comment:
Can you replace variables 's' and 'a' here with something more informative? E.g 'sample' and 'allele'.